.github/workflows/unittests.yml

```diff
@@ -60,6 +60,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
     strategy:
+      fail-fast: false
```

Comment:
@gpotter2 this parameter allows all jobs to run when one fails. This is really useful =)